react.js

```diff
@@ -27,41 +40,47 @@ this.state = {
   dislikes: ["mirrors", "garlic", "wooden stakes"]
 }
 
+var {name, home, dislikes} = this.state
 
+console.log(`#3 - destrucured variables = name: ${name}, home: ${home}, dislikes: ${dislikes} `)
 
 // 4. Write a React method that would add one and update the state of the count each time the method is called.
+console.log("#4 - complete")
 
 this.state = {
   count: 0
 }
 
+updateCount = () => {
+  this.setState({count: 1})
+}
 
```

Comment:
this.setState({count: count ++})
or
this.setState({count: count + 1})
To continue to increment each time the method is called